libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
Can I just remove it and be done with it? I'll have to rewrite the TFA plugins anyway and I think the post-installation messages component has proven to be pretty much of no interest to users.